### ISSUE TITLE
Build error with springloaded 1.2.3.RELEASE

### ISSUE THREAD

**Issue by codacoder** [kensiprell/bootstrap-framework#1]
When I add bootstrap-framework to build.grade per the README, I get the following error...

    org.gradle.api.artifacts.ResolveException: Could not resolve all dependencies for configuration ':agent'.
	at org.gradle.api.internal.artifacts.ivyservice.DefaultLenientConfiguration.rethrowFailure(DefaultLenientConfiguration.java:63)
	at org.gradle.api.internal.artifacts.ivyservice.DefaultResolvedConfiguration.rethrowFailure(DefaultResolvedConfiguration.java:36)
	at org.gradle.api.internal.artifacts.ivyservice.DefaultResolvedConfiguration.getResolvedArtifacts(DefaultResolvedConfiguration.java:59)
    ...
    Caused by: org.gradle.internal.resolve.ModuleVersionNotFoundException: Cannot resolve external dependency org.springframework:springloaded:1.2.3.RELEASE because no repositories are defined.
    Required by:
    qad.app:qad-app:0.1
	at org.gradle.api.internal.artifacts.ivyservice.ivyresolve.NoRepositoriesResolver.resolve(NoRepositoriesResolver.java:48)

And then at the tail of the output I get...

    * What went wrong:
    A problem occurred evaluating root project 'qad-app'.
    > Failed to apply plugin [id 'com.siprell.plugins.bootstrap-framework']
       > /Users/lpeterson/projects/qad-app/build/tmp/bootstrap-v3.3.5.zip (No such file or directory)

I'm new to Grails 3, but I assume the error at the end is being caused by the springloaded error above.  My build.gradle is below. As you can see it's a brand new project, the only changes are those specified in the bootstrap-framework readme plus a few random shots at trying to find the right repository. Any idea on why I'm getting this error?

    buildscript {
        ext {
            grailsVersion = project.grailsVersion
        }
        repositories {
            mavenLocal()
            mavenCentral()
            jcenter()
            maven { url "https://repo.grails.org/grails/core" }
            maven { url "http://dl.bintray.com/kensiprell/gradle-plugins" }
        }
        dependencies {
            classpath "org.grails:grails-gradle-plugin:$grailsVersion"
            classpath 'com.bertramlabs.plugins:asset-pipeline-gradle:2.1.1'
            classpath 'com.siprell.plugins:bootstrap-framework:1.0.2'
        }
    }

    plugins {
        id "io.spring.dependency-management" version "0.5.2.RELEASE"
    }

    version "0.1"
    group "qad.app"

    apply plugin: "spring-boot"
    apply plugin: "war"
    apply plugin: "asset-pipeline"
    apply plugin: 'eclipse'
    apply plugin: 'idea'
    apply plugin: "org.grails.grails-web"
    apply plugin: "org.grails.grails-gsp"
    apply plugin: "com.siprell.plugins.bootstrap-framework"

    ext {
        grailsVersion = project.grailsVersion
        gradleWrapperVersion = project.gradleWrapperVersion
    }

    assets {
        minifyJs = true
        minifyCss = true
    }

    repositories {
        mavenLocal()
        mavenCentral()
        jcenter()
        maven { url "https://repo.grails.org/grails/core" }
    }

    dependencyManagement {
        imports {
            mavenBom "org.grails:grails-bom:$grailsVersion"
        }
        applyMavenExclusions false
    }

    dependencies {
        compile "org.springframework.boot:spring-boot-starter-logging"
        compile "org.springframework.boot:spring-boot-starter-actuator"
        compile "org.springframework.boot:spring-boot-autoconfigure"
        compile "org.springframework.boot:spring-boot-starter-tomcat"
        compile "org.grails:grails-dependencies"
        compile "org.grails:grails-web-boot"

        compile "org.grails.plugins:hibernate"
        compile "org.grails.plugins:cache"
        compile "org.hibernate:hibernate-ehcache"
        compile "org.grails.plugins:scaffolding"
    
        runtime "org.grails.plugins:asset-pipeline"
    

        testCompile "org.grails:grails-plugin-testing"
        testCompile "org.grails.plugins:geb"

        // Note: It is recommended to update to a more robust driver (Chrome, Firefox etc.)
        testRuntime 'org.seleniumhq.selenium:selenium-htmlunit-driver:2.44.0'

        console "org.grails:grails-console"
    }

    task wrapper(type: Wrapper) {
        gradleVersion = gradleWrapperVersion
    }

Edit: forgot to mention I also tried switching the bootstrap-framework version to 1.0.0. I got the same springloaded error but the final error was different:

    * What went wrong:
    A problem occurred evaluating root project 'qad-app'.
    > Plugin with id 'com.siprell.plugins.bootstrap-framework' not found.

More Edit: Should also mention this is grails 3.0.4 with Java 1.8.0_45.

**Comment by kensiprell** [comment]
Lance,

I failed to update the README with the correct version number, which I've now fixed. I'm not sure how you're building your project, and I'm also not sure what the proper procedure is, but this works for me, using Grails 3.0.4 and JDK 1.8.0_45.

    grails create-app test

Add the following lines to ```test/build.gradle``` in the correct locations (README):

    jcenter()
    classpath "com.siprell.plugins:bootstrap-framework:1.0.2"
    apply plugin: "com.siprell.plugins.bootstrap-framework"

And then:

    cd test
    grails compile

I've had varying results using Gradle commands such as ```gradle compileGroovy``` after adding the plugin to a project, but ```grails compile``` always works. Try that and let me know what happens.

Ken
